packages/react/src/components/Card/Card.jsx

```diff
@@ -229,6 +229,7 @@ export const defaultProps = {
   onBlur: undefined,
   tabIndex: undefined,
   testID: CardWrapper.defaultProps.testID,
+  customToolbarContent: undefined,
```

Comment:
Can we add this new props to the docs prop table, too?

Reply:
Added